framework/src/application/genesis_block.ts

```diff
@@ -65,52 +63,44 @@ export interface GenesisAccountStateJSON {
 	};
 }
 
-const accountFromJSON = (account: GenesisAccountStateJSON): GenesisAccountState<AccountAsset> => ({
-	address: Buffer.from(account.address, 'base64'),
-	balance: BigInt(account.balance),
-	nonce: BigInt(account.nonce),
-	keys: {
-		mandatoryKeys: account.keys.mandatoryKeys.map(key => Buffer.from(key, 'base64')),
-		optionalKeys: account.keys.optionalKeys.map(key => Buffer.from(key, 'base64')),
-		numberOfSignatures: account.keys.numberOfSignatures,
-	},
-	asset: {
-		delegate: {
-			...account.asset.delegate,
-			totalVotesReceived: BigInt(account.asset.delegate.totalVotesReceived),
+export const genesisSchema = (accountSchema: object): Schema => ({
```

Comment:
maybe better to call this `genesisBlockSchema` as one it's de-structured it loosed the "genesis block" context.